lib/rubocop/cop/style/method_call_with_args_parentheses.rb

```diff
@@ -61,6 +61,7 @@ module Style
       #   https://bugs.ruby-lang.org/issues/18396.
       # - Parentheses are required in anonymous arguments, keyword arguments
       #   and block passing in Ruby 3.2.
+      # - Parentheses are required when the first or last argument is an endless range.
```

Comment:
Would the following be correct?

```suggestion
      # - Parentheses are required when the first argument is a beginless range or
      #   the last argument is an endless range.
```

Reply:
Yes, that sounds right. Thanks